File: problemas/iniciante/1158/1158.jl
function somaPA(a1, an, n)
  return div((a1 + an) * n, 2)
end

n = parse(Int, readline())
for i in 1:n
  numeros = [parse(Int, x) for x in split(readline())]
  x = numeros[1]
  y = numeros[2]

  x += 1 - (abs(x) % 2)
  println(somaPA(x, x + (2 * (y - 1)), y))
end

File: problemas/iniciante/1158/1158.cs
using System;
using System.Runtime;

class URI {
    static long somaPA(int a0, int n){
        a0 += 1 - (Math.Abs(a0) % 2);
        int an = a0 + (n - 1) * 2;

        return ((long)(a0 + an) * n)/2;
    }

    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine().Trim());
        for(int i = 0; i < N; ++i){
            string[] numeros = Console.ReadLine().Trim().Split(' ');
            int X = int.Parse(numeros[0]);
            int Y = int.Parse(numeros[1]);

            Console.WriteLine(somaPA(X, Y));
        }
    }
}